stdlib/public/core/Builtin.swift

```diff
@@ -816,8 +816,8 @@ func _trueAfterDiagnostics() -> Builtin.Int1 {
 /// of `String.self` (the dynamic type inside the parameter).
 ///
 ///     func printGenericInfo<T>(_ value: T) {
-///         let type = type(of: value)
-///         print("'\(value)' of type '\(type)'")
+///         let tpe = type(of: value)
+///         print("'\(value)' of type '\(tpe)'")
```

Comment:
open to other spellings of the `tpe`, did not find all that much precedence around (saw some `t`, would that be perhaps preferable to `tpe`?)

Reply:
Great catch on this, @ktoso! `t` is preferable to `tpe` — thanks!

Reply:
What do you think about the proposal by @compnerd ?

https://github.com/apple/swift/pull/22575#issuecomment-463388221

> An alternative would be to keep `type` as the variable and use `Swift.type(of:)`.